libraries/Microsoft.Bot.Builder/Adapters/TestFlow.cs

```diff
@@ -165,6 +165,36 @@ public TestFlow AssertReply(string expected, string description = null, uint tim
             return AssertReply(_adapter.MakeActivity(expected), description, timeout);
         }
 
+        /// <summary>
+        /// Adds an assertion that the turn processing logic responds as expected.
+        /// </summary>
+        /// <param name="expected">The part of the expected text of a message from the bot.</param>
+        /// <param name="description">A message to send if the actual response is not as expected.</param>
+        /// <param name="timeout">The amount of time in milliseconds within which a response is expected.</param>
+        /// <returns>A new <see cref="TestFlow"/> object that appends this assertion to the modeled exchange.</returns>
+        /// <remarks>This method does not modify the original <see cref="TestFlow"/> object.</remarks>
+        /// <exception cref="Exception">The bot did not respond as expected.</exception>
+        public TestFlow AssertReplyContain(string expected, string description = null, uint timeout = 3000)
+        {
+            return AssertReply(
+                (reply) =>
+                {
+                    if (!reply.AsMessageActivity().Text.Contains(expected))
```

Comment:
Possible null reference exception.
AsMessageActivity() will return null if the activity is not of ActivityTypes.Message, check the value for null before accessing the text property.

Reply:
This problem also exists in other functions.
Resolved in https://github.com/microsoft/botbuilder-dotnet/pull/1922/commits/55f15b9e9733779ff378e50d193473051172ca21